apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/ReadRowsTest.cs

```diff
@@ -264,28 +255,27 @@ public void ReadRowMissing()
             var tableName = _fixture.TableName;
             var client = _fixture.TableClient;
             var originalIndexes = new Dictionary<BigtableByteString, int>();
-            var rowKeys = new List<BigtableByteString>();
-            for (int i = 0; i < 100; i++)
+            var rowKeys = new BigtableByteString[100];
```

Comment:
Why go from a `List` to an `Array` here? That feels like going backwards.

Reply:
`RowSet.FromRowKeys` takes a params array, so I thought it would be unnecessary to populate a list just to call `ToArray()` on it when I knew the size it was going to be at the start. However, thinking about it again, perhaps the better thing to do is have another overload of `RowSet.FromRowKeys` which takes an `IEnumerable<BigtableByteString>` and have the current overload delegate to it.